cli/alterations/alteration.py

```diff
@@ -17,52 +17,54 @@
 
 from apis import nifiManagment
 from apis import auxiliaryFunctions
-
+import env
 
 def createMerge(nifiConfiguration,information, name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
     merge=auxiliaryFunctions.prepareforAll("./template/alterations/Merge.json",information)
-    merge = auxiliaryFunctions.addSensitiveVariable(merge, "MergeContent", "Maximum Number of Entries", information["maxMessages"])
+    merge = auxiliaryFunctions.addSensitiveVariable(merge, "MergeContent", "Maximum Number of Entries", information[env.ALTERATION_MAX_MESSAGES_TAG])
     nifiConfiguration.create(nameCompose, merge)
-    nifiConfiguration.changeSchedule(nameCompose, "MergeContent", information["windowSeconds"])
+    nifiConfiguration.changeSchedule(nameCompose, "MergeContent", information[env.ALTERATION_WINDOW_SECONDS_TAG])
 
 
 def createDecode(nifiConfiguration,information,name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
     merge=auxiliaryFunctions.prepareforAll("./template/alterations/Encode_Decode.json",information)
```

Comment:
Please, employ descriptive names for the variables, instead of `merge` could be `decode_config`.